<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Minha Pelada</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>451</width>
      <height>411</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Inserir">
     <attribute name="title">
      <string extracomment="Inserir">Inserir</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Nome</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_nome"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Matrícula*</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_mat"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="lbl_numero">
          <property name="text">
           <string>Nº de peladas</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_numero"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btn_inserir">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Inserir</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tbl_dados">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>100</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderShowSortIndicator" stdset="0">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Nome</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Matrícula</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Nº de peladas</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Cargo</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="btn_ordenar_nome">
          <property name="text">
           <string>Ordenar por Nome</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_ordenar_ass">
          <property name="text">
           <string>Ordenar por Assiduidade</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_ordenar_prio">
          <property name="text">
           <string>Ordenar por Prioridade</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Estatísticas</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>421</width>
        <height>361</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="gb_avg">
         <property name="title">
          <string>Porcentagem de Servidores</string>
         </property>
         <widget class="QLabel" name="lbl_serv">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>411</width>
            <height>61</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QGroupBox" name="gb_highest">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>60</y>
            <width>419</width>
            <height>61</height>
           </rect>
          </property>
          <property name="title">
           <string>Porcentagem de Dependentes</string>
          </property>
          <widget class="QLabel" name="lbl_dep">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>10</y>
             <width>411</width>
             <height>51</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </widget>
         <widget class="QGroupBox" name="gb_lowest">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>130</y>
            <width>419</width>
            <height>61</height>
           </rect>
          </property>
          <property name="title">
           <string>Porcentagem de Convidados</string>
          </property>
          <widget class="QLabel" name="lbl_conv">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>10</y>
             <width>411</width>
             <height>51</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </widget>
         <widget class="QGroupBox" name="gp_mais_ass">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>200</y>
            <width>421</width>
            <height>71</height>
           </rect>
          </property>
          <property name="title">
           <string>Peladeiro mais assíduo</string>
          </property>
          <widget class="QLabel" name="lbl_mais_ass">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>-1</y>
             <width>421</width>
             <height>71</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </widget>
         <widget class="QGroupBox" name="gp_menos_ass">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>280</y>
            <width>421</width>
            <height>71</height>
           </rect>
          </property>
          <property name="title">
           <string>Peladeiro menos assíduo</string>
          </property>
          <widget class="QLabel" name="lbl_menos_ass">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>421</width>
             <height>81</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>410</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>* '0' para convidado</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>456</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionSalvar"/>
    <addaction name="actionCarregar"/>
   </widget>
   <addaction name="menuArquivo"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCarregar">
   <property name="text">
    <string>Carregar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
